src/routes/account/+page.server.ts

```diff
@@ -0,0 +1,107 @@
+import type { Actions, PageServerLoad } from './$types';
+import { error, redirect } from '@sveltejs/kit';
+import {
+  getStripeCustomerWithSubscriptions,
+  getBlendProSubscription,
+  getCustomerPortalSession,
+  createStripeSession,
+  isSubscribedToBlendPro,
+  isOrganizationMember,
+} from '$lib/server/subscriptionUtils';
+import { checkSessionAuth, getUserData, getUserOrganizations, isUserGlobalAdmin, readPath, writePath } from '$lib/server/firebaseUtils';
+
+let email = '';
+let name = '';
+let uid = '';
```

Comment:
Why are these here? I think this scoping will break things when multiple users hit the functions.